### PR TITLE
feat: dark color for code block in dark mode

### LINKED ISSUE #852: Code block should have dark colors too
Code block from markdown have grey background and black text even on dark theme. We should use dark theme colors vars instead to make it fit better. See example here

https://snapshot.org/#/strategy/contract-call

Screenshot_20211009-113227_Chrome

### PR DESCRIPTION
**Pull request by sarmatdev**
Fixes #852 .

Changes proposed in this pull request:
- Fix code block color in dark mode
- Add 1px border for code block in dark mode for contrast and comfortable viewing
-

### CODE REVIEW

**Comment by samuveth** [comment]
Great @sarmatdev, can you also do the same for tables while your at it?
image

File: src/components/Ui/Markdown.vue [hunk 1]
```diff
@@ -353,7 +353,8 @@ const markdown = computed(() => {
   word-wrap: normal;
 }
 .dark-mode .markdown-body pre {
-  color: var(--bg-color);
+  background-color: var(--bg-color);
+  border: 1px solid var(--border-color);
```
[the remark this answers is not in the document]
Comment (sarmatdev) [reply src/components/Ui/Markdown.vue:357]: Changed background color to the --border-color, but text in there looks hard to read with --text-color.
First image with default --text-color text styling, second with --heading-color(which is plain white color)

Which one is good?

Снимок экрана 2021-10-09 в 13 05 41
Снимок экрана 2021-10-09 в 13 05 49

[the remark this answers is not in the document]
Comment (bonustrack) [reply src/components/Ui/Markdown.vue:357]: @sarmatdev Second one is fine. Could you reduce the font-size a bit for code block also?

**Comment by sarmatdev** [comment]
> Looks good. @sarmatdev should I open another issue for the table or will you add that here?

Another issue and new branch will be good. But I can't debug this tables, in dev mode I could not find your test proposal🤔

**Review by samuveth — APPROVED** [review]
LGTM

**Pull request merged**